ts/Dashboards/Plugins/KPIComponent.ts

```diff
@@ -314,6 +318,8 @@ class KPIComponent extends Component {
         this.contentElement.style.display = 'flex';
         this.contentElement.style.flexDirection = 'column';
 
+        this.setChartValue();
```

Comment:
I tested it and came to the conclusion that it's better to execute this method in `load`. When updating the component, `setValue` and `render` are called at once, so the chart would be updated too many times unnecessarily. Also, I can't do it only in `render`, because the extremes sync executes only `setValue`, without `render`.